README.md

```diff
@@ -61,21 +61,26 @@ Download the lastest version of Android Studio at the following link: http://dev
 Run the app by clicking Run on Android Studio. Choose any Android Device as your emulator and the application should start. You will be prompted with a welcome screen, after which you are invited to type in your name and a welcome message will be displayed. Next screen is a simple calculator you can play around with. Following is the quiz itself, which contains checkboxes for the types of conversions (ie. Decimal to Binary or Octal to Hexadecimal) and radio buttons for the length of the quiz (ie. 6 questions, 12 questions, etc.) You will then be prompted with the quiz interface itself, where a number is given and the user will be prompted to type in the correct conversion. A "Change Number" feature is added if the current problem is too difficult. A "Percentage" feature has also been implemented to tell the user how well they are doing as they are progressing throughout the quiz. 
 
 ## Javadoc
-JavaDoc Link: http://ucsb-cs56-projects.github.io/cs56-android-conversion-showdown/javadoc/
+JavaDoc Link: https://github.com/kerrymo/cs56-android-conversion-showdown/tree/proj01/javadoc
+
+
```

Comment:
Please revert this change - this is supposed to be a link to the rendered html page, not to the github file listing.

Reply:
Fixed